src/datalinker/config.py

```diff
@@ -67,36 +131,92 @@ class Config(BaseSettings):
                 "This URL prefix is used for the IVOA DataLink API and for"
                 " any other helper APIs exposed via DataLink descriptors"
             ),
+            validation_alias="pathPrefix",
         ),
     ] = "/api/datalink"
 
     profile: Annotated[
         Profile,
-        Field(title="Application logging profile"),
+        Field(title="Application logging profile", validation_alias="profile"),
```

Comment:
You don't need to set a validation alias unless the camel-case and snake-case forms are different, although of course it doesn't hurt anything to do so.